samples/snippets/src/main/java/com/example/spanner/HubbleTransactionsCodeLab.java

```diff
@@ -14,21 +14,24 @@
 
   private static final String SUBJECT = "Random subject for the email for testing purpose %s";
 
+  private static final int NUM_PROCESSORS = Runtime.getRuntime().availableProcessors();
+  private static final ExecutorService EXECUTOR_SERVICE =
```

Comment:
The way we are using it will not create issues as we are calling each function separately and every time all classes will get loaded and new object will get created. 
In case someone want to call more than one functions(contains thread logic) from same object then this will give an issue.
For keeping our self safe from future and keeping its implementation safe I will modify it little bit to make it work in all the way